src/sentry/tsdb/base.py

```diff
@@ -546,7 +546,7 @@ def get_distinct_counts_totals(
         jitter_value: int | None = None,
         tenant_ids: dict[str, int | str] | None = None,
         referrer_suffix: str | None = None,
-        conditions: list[tuple[str, str, str]] | None = None,
+        conditions: list[tuple[str, str, Any]] | None = None,
```

Comment:
Why are these changing from `str` -> `any`? Is it possible to further type this?

Reply:
the return type for that value is `... | Any | ...`, not sure why it passed mypy in the first place but `Any` should encompass all those types

Reply:
Are you sure we don't know the specific type here? Are these basically `SnubaConditions` or something similar? It'd be better to have the typing correct, if possible.

Reply:
yes it's SnubaCondition

Reply:
If it's a `SnubaCondition`, can we use that type?